phase0/CRC Design/GameRenderer.txt

```diff
@@ -1,6 +1,7 @@
 Class name: GameRenderer
 Parent class: 
-Subclasses: 
+Subclasses:
+Architecture Section: Controller
```

Comment:
I think Game renderer should be under GUI cuz it just displays the scenes and allows input

Reply:
yeah i think game renderer is GUI